spswarehouse/powerschool/calpads.py

```diff
@@ -0,0 +1,220 @@
+import time
+from spswarehouse.powerschool.powerschool import (
+                                            navigate_to_specific_state_report, 
+                                            download_latest_report_from_report_queue_reportworks, 
+                                            download_latest_report_from_report_queue_system,
+                                            switch_to_school,
+                                            powerschool_report_helper_type_in_element_by_id,
+                                            powerschool_report_helper_type_in_element_by_name,
+                                            powerschool_report_helper_select_visible_text_in_element_by_id,
+                                            powerschool_report_helper_select_visible_text_in_element_by_name,
+                                            powerschool_report_helper_click_element_by_id,
+                                            )
+
+from selenium.webdriver.chrome.webdriver import WebDriver
+
+def download_calpads_report_for_school(driver: WebDriver, school_full_name: str, report_name: str, school_subdistrict_name: str, 
+                                       file_postfix: str, destination_directory_path: str, report_parameters: dict, validation_only_run: bool=False):
+    """
+    Switches to the desired school in PowerSchool and calls the function to generate the desired
+    report. Note: This function currently only supports EOY reports. Fall 1 and Fall 2 reports
+    should not use this function until it is expanded to differentiate based on the CALPADS
+    submission window.
+    """
+
+    # The SCSC report needs to be run from the District Office level in order to properly generate LEA IDs without dropping leading zeros
+    if report_name == "Student Course Section Records":
+        switch_to_school(driver, 'District Office')
+    else:
+        switch_to_school(driver, school_full_name)
+
+    if(report_name == "Student Incident Records (SINC)"):
+        return download_calpads_report_for_school_student_incident_records_sinc(
+                                                                    driver=driver, 
```

Comment:
Is this the default indent? I don't personally see the value in having these indented so far.

Reply:
VSCode does default to having parameters align with the left parentheses. I'm committing a change to have them all start just tabbed over once from the line before.